edk2toollib/uefi/edk2/path_utilities.py

```diff
@@ -97,7 +97,11 @@ def GetEdk2RelativePathFromAbsolutePath(self, abspath):
 
         package = self.GetContainingPackage(abspath)
         if package is not None:
-            relpath = abspath[str(temp_path).find(package):]
+            if f'{package.upper()}.DEC' in str(temp_path).upper():
```

Comment:
maybe i am dense but can you write out in comments what you are trying to account for here.   I think I might need comments written out for the previous change too which included the temp_path stuff above and slicing based on temp_path.